yambo-back/models/cardModel.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const cardSchema = new Schema({
+  front: {
+    type: String,
+    required: true,
+    index: true,
+  },
+  back: {
```

Comment:
I think card back should be set up as an array of objects, so that the public API call will be able to populate different fields on the card back. At minimum to start I think the back property should look like:

back: [{kana: String}, {definition: String}]

To start, the public API call would populate both these fields. If the user is creating their own card, the kana field can be left blank. In the future, additional fields may be added.

Reply:
I was taking into account that because the current setup is two textareas, we'd just input two basic string inputs. If we'd hard-code kana and definition objects, then custom user input would be a bit complicated as of right now. 

The good point of using textareas is their native implementation of line-breaks with regex. Textareas inserts `\n` with each line-break from the input box into the output string. Thus if we do automatic lookup and stringify it all like 'Kana: えらい\nDefinition: amazing' and regardless if a string has the regex \n within, the JSX and output it accordingly. See my previous project here for how to output: https://github.com/KeroseneSlickback/yakumon-frontend/blob/main/src/Pages/Store.tsx#L150

Reply:
Understood, moving forward with single string values for front and back side of card. As to the need for an array of objects to store metadata, TBD.